src/index.html

```diff
@@ -5,10 +5,29 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>Moyo header v1.1</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="headr">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="MOYO logo">
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="liItem"><a href="#apple" class="link is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="liItem">
            <a href="#apple" class="link is-active">Apple</a>
          </li>
```
Новый тег - новая строка